travis-build/Makefile

```diff
@@ -53,7 +53,7 @@ ${LOCAL_DIR}/bin/pd-server:
 	cp bin/pd-server ${LOCAL_DIR}/bin/pd-server
 
 prepare_rust:
-	sh ~/rust/lib/rustlib/uninstall.sh && sh ~/rust-installer/rustup.sh --date=2016-09-05 --prefix=~/rust --disable-sudo --channel=nightly
+	sh ~/rust/lib/rustlib/uninstall.sh && sh ~/rust-installer/rustup.sh --date=2016-09-10 --prefix=~/rust --disable-sudo --channel=nightly
```

Comment:
I upgraded clippy to `0.0.90`, `rustc 09-05` can't compile it. Are there any problems of upgrading？Or should I roll clippy back?